app/helpers/vacancies_helper.rb

```diff
@@ -212,4 +212,32 @@ def referrer_organisation_slug(referrer)
 
     organisation_slug
   end
+
+  def vacancy_statistics_bar_chart(referrer_counts)
+    text_style = {
+      color: "#000000",
+      fontSize: 20,
+      fontName: "GDS Transport",
+    }
+
+    # sort with highest value on top - using a hash results in an arbitrary ordering
+    bar_chart sort_referrer_counts(referrer_counts),
+              library: {
+                hAxis: {
+                  textStyle: text_style,
+                },
+                tooltip: {
+                  textStyle: text_style,
+                },
+                vAxis: {
+                  textStyle: text_style,
+                },
+              },
+              height: "#{referrer_counts.size * 60}px",
+              colors: ["#003764"]
```

Comment:
Could we set this to the GovUK "official blue"? `#1d70b8`. Used for both ` govuk-colour("blue")` , `$govuk-brand-colour` and `$govuk-link-colour`:  https://design-system.service.gov.uk/styles/colour/